instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/instrumentation/webflux/SpringWebfluxInstrumentationAutoConfiguration.java

```diff
@@ -28,10 +28,8 @@
 @Configuration
 public class SpringWebfluxInstrumentationAutoConfiguration {
 
-  public SpringWebfluxInstrumentationAutoConfiguration() {}
-
   @Bean
-  static WebClientBeanPostProcessor otelWebClientBeanPostProcessor(
+  WebClientBeanPostProcessor otelWebClientBeanPostProcessor(
```

Comment:
will this start causing warnings once https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11185 is merged?